src/hex.ts

```diff
@@ -55,6 +59,16 @@ export function assertIsStrictHexString(value: unknown): asserts value is Hex {
   );
 }
 
+/**
+ * Validates that the passed prefixed hex string is a correct hex address.
```

Comment:
Nit: maybe more accurate to use the word "valid" here rather than 'correct"
```suggestion
 * Validates that the passed prefixed hex string is a valid hex address.
```